charts/metallb/values.yaml

```diff
@@ -225,3 +225,15 @@ speaker:
     periodSeconds: 10
     successThreshold: 1
     timeoutSeconds: 1
+
+# frr contains configuration specific to the MetalLB FRR container.
+frr:
```

Comment:
This should probably be moved from `.Values.frr` to `.Values.speaker.frr`, as it is a configuration specifically for the speaker.

Reply:
you are right. i will fix it. thanks